src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (radius * radius) * Math.PI;
+    }
+
+    @Override
+    public String getColor() {
+        return super.getColor();
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: Circle, area: "
+                + String.format("%.2f", calculateArea())
+                + " sq. units, radius: "
+                + String.format("%.0f", radius)
+                + " units, color: "
+                + getColor();
+    }
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/034c21a3-e1cf-4234-9989-f3bc954cd13e)
```suggestion
    @Override
    public String draw() {
        return "Figure: Circle, area: "
                + String.format("%.2f", calculateArea())
                + " sq. units, radius: "
                + String.format("%.0f", radius)
                + " units, color: "
                + getColor();
    }
```